src/index.js

```diff
@@ -32,9 +33,8 @@ cytoscape.use = function( ext ){
 };
 
 // replaced by build system
-cytoscape.version = require('./version');
+cytoscape.version = version;
 
 // expose public apis (mostly for extensions)
-cytoscape.stylesheet = cytoscape.Stylesheet = Stylesheet;
```

Comment:
It looks like this line was missed

Reply:
When I revise this line, I get the following in the debug page while `npm run watch`ing:

```
Uncaught TypeError: Cannot read property 'finiteLinesIntersect' of undefined
```